src/openms/include/OpenMS/CHEMISTRY/EnzymaticDigestion.h

```diff
@@ -112,7 +112,7 @@ namespace OpenMS
     void digestUnmodified(const StringView& sequence, std::vector<StringView>& output, Size min_length = 1, Size max_length = 0) const;
 
     /// Returns true if the fragment starting at position @p pos with length @p length within the sequence @p sequence could be generated by the current model
-    bool isValidProduct(const String& sequence, Size pos, Size length, bool ignore_missed_cleavages = true) const;
+    bool isValidProduct(const String& sequence, Size pos, Size length, bool ignore_missed_cleavages = true, bool allow_random_asp_pro_cleavage = false) const;
```

Comment:
this should not be in the base class of RNases and Proteases

Reply:
Ok, we discussed some changes for this offline. Wait for update.